spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -1631,4 +1631,16 @@ An `InterceptionFactory` can be obtain be calling `BeanManager.createInterceptio
 
 The container must provide a bean with scope `@Dependent`, bean type `InterceptionFactory` and qualifier `@Default` allowing producer methods to obtain this factory to produce their instance.
 
+Following example demonstrates producer method definition using `InterceptionFactory`. Interceptor using `ActionBinding` interceptor binding will be associated with produced bean:
```

Comment:
Thanks for suggestions. Maybe following would be better:
`Produced bean will have a single interceptor associated by ActionBinding`, wdyt?